go/libraries/doltcore/sqle/reflog_table_function.go

```diff
@@ -204,20 +195,28 @@ func (rltf *ReflogTableFunction) IsReadOnly() bool {
 }
 
 func (rltf *ReflogTableFunction) Expressions() []sql.Expression {
-	if rltf.refExpr != nil {
-		return []sql.Expression{rltf.refExpr}
-	}
 	return []sql.Expression{}
 }
```

Comment:
This is a little bit unsafe – we need to keep the expressions around so that they can be processed correctly...

These query plan trees will go through the analyzer and get modified and as expressions/nodes are processed, they will be replaced with new instances. If we aren't able to get the expressions, then we can't accurately create a new instance and the argument will likely drop out and the function may end up running differently than what the user requested.